packages/node_modules/brightspace-auth-keys-dynamodb-store/src/public-key-table-setup.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+function createPublicKeyTable(db, tablename, readCapacity, writeCapacity) {
+	var params = {
+		AttributeDefinitions: [
+			{
+				AttributeName: 'PublicKey',
+				AttributeType: 'S'
+			}
+		],
+		KeySchema: [
+			{
+				AttributeName: 'PublicKey',
+				KeyType: 'HASH'
+			}
+		],
+		ProvisionedThroughput: {
+			ReadCapacityUnits: readCapacity || 5,
+			WriteCapacityUnits: writeCapacity || 5
+		},
+		TableName: tablename
+	};
+
+	return db.createTable(params).promise().then(() => {
+		return setTtl(db, tablename);
+	}).catch((e) => {
+		// ResourceInUseException indicates the table already exists
+		if (e.code !== 'ResourceInUseException') {
+			throw e;
+		}
+	});
+}
+
+function setTtl(db, tablename) {
+	var params = {
+		TableName: tablename,
+		TimeToLiveSpecification: {
+			AttributeName: 'ExpiresAt',
+			Enabled: true
+		}
+	};
+	return db.updateTimeToLive(params).promise().catch(e => {
+		// UnknownOperationException will be thrown for Dynamo Local

```

Comment:
Is there a way to "detect" dynamo local, so we can only ignore this for dynamo local?

Reply:
The UnknownOperationException exception won't happen unless it doesn't support TTL so even if it was production that threw it I would want to ignore it.